korman/ui/modifiers/render.py

```diff
@@ -168,3 +168,30 @@ def visregion(modifier, layout, context):
 
     # Other settings
     layout.prop(modifier, "replace_normal")
+
+
+def fogenv(modifier, layout, context):
+    row = layout.row()
+    row.label("Environment")
+    row.template_ID(modifier, "fog_env", new="world.new")
+
+    if modifier.fog_env:
+        box = layout.box()
+        split = box.split()
+
+        fni = modifier.fog_env.plasma_fni
+
+        col = split.column()
+        col.prop(fni, "fog_method")
+        col.prop(fni, 'fog_color')
+
+        col = split.column()
+        row = col.row()
+        row.active = fni.fog_method == "linear"
```

Comment:
Korman prefers to hide unused properties.

Reply:
Hiding the `start` field caused some annoying visual jumping of the `end`/`density` fields when toggling between fog types. I tried to work around it by adding a blank space if we didn't need the `start` field, but apparently Blender ignores that :\